easybuild/easyconfigs/h/hipify-clang/hipify-clang-4.2.0-gcccuda-2020b.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'CMakeMake'
+
+name = 'hipify-clang'
+version = '4.2.0'
+
+homepage = 'https://github.com/ROCm-Developer-Tools/HIPIFY'
+description = """Hipify-clang is a clang-based tool for translating CUDA
+  sources into HIP sources. It translates CUDA source into an abstract syntax
+  tree, which is traversed by transformation matchers. After applying all the
+  matchers, the output HIP source is produced."""
+
+toolchain = {'name': 'gcccuda', 'version': '2020b'}
+
+source_urls = ['https://github.com/ROCm-Developer-Tools/HIPIFY/archive/refs/tags/']
+sources = ['rocm-%(version)s.tar.gz']
+checksums = ['afdc82ae00e14e8e742be6cd47d8fb120d18fc52fe96cba8d8ac4c56176a432e']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('zlib', '1.2.11'),
+]
+
+dependencies = [
+    # ('cuDNN', '8.0.4.30', '-CUDA-11.1.1', True),
+    ('Clang', '11.0.1'),
+    # ('Python', '3.8.6'),
```

Comment:
I would keep the commented out dependencies together, and maybe even comment them out but without running the tests until we have `llvm-lit` available as a build dep?